src/chatdlg.cpp

```diff
@@ -24,30 +24,32 @@
 
 #include "chatdlg.h"
 
-
 /* Implementation *************************************************************/
 CChatDlg::CChatDlg ( QWidget* parent ) :
-    CBaseDlg ( parent, Qt::Window ) // use Qt::Window to get min/max window buttons
+    CBaseDlg ( parent,
```

Comment:
Weird one...  Ideally should remain as was.

Reply:
I guess it would with the larger line limit.